tests/unit/transformers/test_numerical.py

```diff
@@ -421,12 +421,14 @@ def test__reverse_transform_rounding_none_with_nulls(self):
         - First column of the input array as entered, replacing the indicated value with a Nan.
         """
         # Setup
-        data = pd.DataFrame([
```

Comment:
I feel like this syntax show up a lot in our code. Why is this the only one that's changed?

Reply:
It's because of the `columns=['a', 'b']` they are written in one line and the others are splitted.
When we have a look we usually tend to visually approve that

```
data = pd.DataFrame({
        ...
        ...
        ...
        columns=['a', 'b'],
 })
 ```
 But this is actually becoming a inconsistent usage of the `[]` since you have array which it's values are splitted in multiple lines and another that is one-liner. So if we had no `columns` it would be fine.